pkgs/top-level/all-packages.nix

```diff
@@ -20569,7 +20569,7 @@ with pkgs;
     boost = boost160;
   };
 
-  librepcb = libsForQt5.callPackage ../applications/science/electronics/librepcb { };
+  librepcb = libsForQt59.callPackage ../applications/science/electronics/librepcb { };
```

Comment:
Why is it necessary to use the older qt 5.9 instead of the current default 5.11 ?

Reply:
Thanks for the input!
I just tried 5.11 and it works too.